pkg/cmd/server/server.go

```diff
@@ -18,11 +18,16 @@ package server
 
 import (
 	"context"
+	"errors"
 	"fmt"
+	"io/ioutil"
+	"os"
 	"reflect"
 	"sync"
 	"time"
 
+	"golang.org/x/oauth2/google"
+
```

Comment:
I'm going to change my stance on imports. I don't want us to spend time thinking about them. If the code compiles, that's good enough for me, regardless of order, whitespace, grouping, etc.